src/main/kotlin/org/jitsi/jibri/service/impl/FileRecordingJibriService.kt

```diff
@@ -118,8 +119,11 @@ class FileRecordingJibriService(private val fileRecordingParams: FileRecordingPa
     private var processMonitorTask: ScheduledFuture<*>? = null
 
     init {
+        if (!Files.isWritable(fileRecordingParams.recordingDirectory)) {
```

Comment:
WritableDirectory used to contain logic to create the directory if it didn't exist. Is it normal that this is missing here? 

Also is "Files.isWritable" a sufficient check for a directory (since we need execute permissions on a directory to effectively list its contents or add a file to it)?